client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -165,12 +165,14 @@ class TaxonomyManagerListItem extends Component {
 					) }
 				</span>
 				{ typeof term.post_count !== 'undefined' && (
-					<Count
-						ref="count"
-						count={ term.post_count }
-						onMouseEnter={ this.showTooltip }
-						onMouseLeave={ this.hideTooltip }
-					/>
+					<div className="taxonomy-manager__count">
```

Comment:
Is it possible to apply the class to the `Count` component directly rather than adding more markup? Just curious, not a blocker :)

Reply:
I think that because of the `display: flex` on `.taxonomy-manager__item`, the `Count` component needs to be wrapped on in a div to resolve the misalignment - I couldn't really think of an alternative, unfortunately!